static-assets/themes/cstudioTheme/css/console.css

```diff
@@ -116,6 +116,15 @@ input[type="checkbox"].content-type-selector--checkbox {
   margin-left: 239px;
 }
 
+.embedded #admin-console #categories-panel{
+  display: none;
+}
+
+.embedded #admin-console #cstudio-admin-console-workarea {
+  width: 100%;
+  margin-left: 0;
+}
+
```

Comment:
There's a big margin at the top when the toolbar is removed. I don't see any styles addressing that; are you addressing somewhere else?